tests/unit/test_estimates.py

```
@@ -0,0 +1,235 @@
+        # Retrieve
+        retrieved = client.estimates.retrieve(workspace_slug, project.id)
+        assert retrieved is not None
+        assert isinstance(retrieved, Estimate)
+        assert retrieved.id == estimate.id
+        assert retrieved.name == estimate.name
+
+        # Cleanup
+        try:
+            client.estimates.delete(workspace_slug, project.id)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)